interface/patient_file/encounter/forms.php

```diff
@@ -437,30 +427,30 @@ function divtoggle(spanid, divid) {
 
 <!-- Get the documents tagged to this encounter and display the links and notes as the tooltip -->
 <?php 
-	$docs_list = getDocumentsByEncounter($pid,$_SESSION['encounter']);
-	if(count($docs_list) > 0 ) {
+    $docs_list = getDocumentsByEncounter($pid,$_SESSION['encounter']);
+    if(count($docs_list) > 0 ) {
 ?>
 <div class='enc_docs'>
 <span class="bold"><?php echo xlt("Document(s)"); ?>:</span>
 <?php
-	$doc = new C_Document();
```

Comment:
Whitespace is killing me.  Please undertake to process the whole damn application if you are removing tabs.  Atom editor and probably Eclipse are killlllllling us here.  We need to get this done before GSOC.